cmd/backend.go

```diff
@@ -56,7 +62,7 @@ func main() {
 
 	// Handling INTERRUPT signal for cleanup in a new goroutine.
 	// This is not necessary, but good for log keeping
-	c := make(chan os.Signal)
+	c := make(chan os.Signal, 1)
```

Comment:
missed this, channels should be deferred closed.

Reply:
Done.